src/nvc.rs

```diff
@@ -460,6 +569,24 @@ impl Nvc {
                         OPCODE_SYSTEM_REGISTER_ID_PSW => self.set_reg_psw(value),
                         OPCODE_SYSTEM_REGISTER_ID_CHCW => {
                             logln!("WARNING: ldsr chcw not yet implemented (value: 0x{:08x})", value);
```

Comment:
nit: Should probably write "not fully implemented" to indicate parts of this register are implemented